outbox-spring/src/main/java/io/praesidio/outbox/SpringScheduler.java

```diff
@@ -0,0 +1,15 @@
+package io.praesidio.outbox;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+
+@RequiredArgsConstructor
+class SpringScheduler {
+
+    private final MessageRelay messageRelay;
+
+    @Scheduled
```

Comment:
From javadoc: `Annotation that marks a method to be scheduled. Exactly one of the cron, fixedDelay, or fixedRate attributes must be specified.`